Examples/Scotty-tvOSExample/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  Scotty-tvOSExample
+//
+//  Created by Fernando Arocho on 1/8/19.
+//  Copyright © 2019 Bottle Rocket Studios. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        return true
+    }
+
+    func applicationWillResignActive(_ application: UIApplication) {
```

Comment:
the methods that are not used can be removed. this is similar to how the iOS Example in Scotty looks